src/main/java/com/rcv/ResultsWriter.java

```diff
@@ -56,6 +56,8 @@
   private static final String CDF_GPU_ID = "gpu-001";
   private static final String CDF_REPORTING_DEVICE_ID = "rd-001";
 
+  private static final Map<String, String> candidateCodeToJsonId = new HashMap<>();
```

Comment:
Would it be clearer to change JsonId => CdfId ?   Here and elsewhere...

Reply:
Sure.